src/components/PeoplePage.tsx

```diff
@@ -1,35 +1,124 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
 import { Loader } from './Loader';
+import { PeopleFilters } from './PeopleFilters';
 import { PeopleTable } from './PeopleTable';
 
 export const PeoplePage = () => {
-  return (
-    <>
-      <h1 className="title">People Page</h1>
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setError] = useState(false);
+  const [searchParams] = useSearchParams();
 
-      <div className="block">
-        <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(result => {
+        const peopleFromServer = result.map(pep => ({ ...pep }));
+
+        peopleFromServer.forEach(person => {
+          Object.assign(person, {
+            mother: peopleFromServer.find(m => m.name === person.motherName)
+                || null,
+            father: peopleFromServer.find(f => f.name === person.fatherName)
+                || null,
+          });
+        });
```

Comment:
```suggestion
        const peopleFromServer = result.map(person => ({
          ...person,
          mother: peopleFromServer.find(m => m.name === person.motherName)
                || null,
            father: peopleFromServer.find(f => f.name === person.fatherName)
                || null
        }));
```
we can simplify and avoid additional iteration